tests/driver_isl29125/Makefile

```diff
@@ -0,0 +1,22 @@
+APPLICATION = driver_isl29125
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_i2c
+
+USEMODULE += isl29125
+USEMODULE += vtimer
+
+ifneq (,$(TEST_ISL29125_I2C))
+  CFLAGS += -DTEST_ISL29125_I2C=$(TEST_ISL29125_I2C)
+else
+  # set random default
+  CFLAGS += -DTEST_ISL29125_I2C=I2C_0
+endif
+ifneq (,$(TEST_ISL29125_IRQ_PIN))
+  CFLAGS += -DTEST_ISL29125_IRQ_PIN=$(TEST_ISL29125_IRQ_PIN)
+else
+  # set random default
+  CFLAGS += -DTEST_ISL29125_IRQ_PIN=GPIO_5
```

Comment:
Uhm, yep, missing in the initializer, but part of the documentation (implicitly).